model/contract/UsageControlComponentEnpoint.ttl

```diff
@@ -0,0 +1,44 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix ids: <https://w3id.org/idsa/core/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+# Classes
+# -------
+
+ids:UsageControlComponentEndpoint
+    a owl:Class;
+    rdfs:label "Usage Control Component Endpoint class"@en ;
+    rdfs:comment "Defined usage control component endpoints (i.e. PIP, PXP, etc.) in the IDS."@en;
+	.
+
+ids:PxpEndpoint
+    a owl:Class;
+    rdfs:subClassOf ids:UsageControlComponentEndpoint;
+    rdfs:label "pxp endpoint"@en;
+    rdfs:comment "The reference to the endpoint which operates a specific action, URI or reference to an associated PXP interface."@en;
```

Comment:
Grammar/style: please write "the endpoint that", because "the endpoint" in itself is not sufficiently defined. See https://www.grammarly.com/blog/which-vs-that/

Reply:
Corrected.